src/apps/relay/netengine.c

```diff
@@ -350,6 +350,10 @@ void set_ssl_ctx(ioa_engine_handle e, turn_params_t *params) {
     params->tls_ctx_update_ev = ev;
     TURN_MUTEX_UNLOCK(&turn_params.tls_mutex);
   }
+  else
+  {
+    if (args) free(args);
```

Comment:
you'll need to add braces to this if statement to pass the linter.

Reply:
Done!